roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   hostNetwork: true
 {% if kube_version | version_compare('v1.6', '>=')  %}
-  dnsPolicy: ClusterFirstWithHostNet
+  dnsPolicy: ClusterFirst
```

Comment:
This effects in falling down to default value, same as for <v1.6, so maybe there would be more optimal to completely remove enclosing test and this setting?